analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RuleChecks/EmptyCollectionsShouldNotBeEnumeratedBase.cs

```diff
@@ -126,14 +126,6 @@ protected override ProgramState PreProcessSimple(SymbolicContext context)
         {
             return ProcessAddMethod(context.State, methodReference.Method, methodReference.Instance) ?? context.State;
```

Comment:
There are some .NET ITs failing.
The reason is that for the methods that are NOT activating this rule, we set _some_ of the constraints this rule used to set.

After this+invocation PR is merged, we should be good to go.